modules/sdk-test/src/bitgo/lib/test_bitgo.ts

```diff
@@ -0,0 +1,624 @@
+      },
+      admin: {
+        policy: {
+          id: '598f606cd8fc24710d2ebadda4b955fb',
+          version: 0,
+          date: '2017-08-12T20:09:16.472Z',
+          rules: [],
+        },
+      },
+      clientFlags: [],
+      allowBackupKeySigning: false,
+      balanceString: '10000000000000000000',
+      confirmedBalanceString: '10000000000000000000',
+      spendableBalanceString: '10000000000000000000',
+      receiveAddress: {
+        id: '5a8453c23fd075b907574338a878f4fb',
+        address: '0xa7f9ca5c1268b0082db1833d30f33d3cfd4286d8',
+        chain: 0,
+        index: 701,
+        coin: 'gteth',
+        lastNonce: 0,
+        wallet: '598f606cd8fc24710d2ebadb1d9459bb',
+        coinSpecific: {
+          nonce: -1,
+          updateTime: '2018-02-14T15:20:34.188Z',
+          txCount: 0,
+          pendingChainInitialization: false,
+          creationFailure: [],
+        },
+      },
+      pendingApprovals: [],
+    };
+
+    const wallet = new Wallet(this, this.coin('gteth'), walletData);
+
+    // Nock calls to platform for building transactions and getting user key
+    // Should be OK to persist these since they are wallet specific data reads
+    nock(this._baseUrl)
+      .persist()
+      .filteringRequestBody(() => '*')
+      .post(`/api/v2/gteth/wallet/${wallet.id()}/tx/build`, '*')
+      .reply(200, {
+        gasLimit: 500000,
+        gasPrice: 20000000000,
+        nextContractSequenceId: 101,
+      })
+      .get(`/api/v2/gteth/key/${wallet.keyIds()[KeyIndices.USER]}`)
+      .reply(200, {
+        id: '598f606cd8fc24710d2ebad89dce86c2',
+        users: ['543c11ed356d00cb7600000b98794503'],
+        pub: 'xpub661MyMwAqRbcFXDcWD2vxuebcT1ZpTF4Vke6qmMW8yzddwNYpAPjvYEEL5jLfyYXW2fuxtAxY8TgjPUJLcf1C8qz9N6VgZxArKX4EwB8rH5',
+        ethAddress: '0x26a163ba9739529720c0914c583865dec0d37278',
+        encryptedPrv:
+          '{"iv":"15FsbDVI1zG9OggD8YX+Hg==","v":1,"iter":10000,"ks":256,"ts":64,"mode":"ccm","adata":"","cipher":"aes","salt":"hHbNH3Sz/aU=","ct":"WoNVKz7afiRxXI2w/YkzMdMyoQg/B15u1Q8aQgi96jJZ9wk6TIaSEc6bXFH3AHzD9MdJCWJQUpRhoQc/rgytcn69scPTjKeeyVMElGCxZdFVS/psQcNE+lue3//2Zlxj+6t1NkvYO+8yAezSMRBK5OdftXEjNQI="}',
+      });
+
+    const params: any = {
+      module: 'account',
+      action: 'tokenbalance',
+      contractaddress: BitGo.V2.TEST_ERC20_TOKEN_ADDRESS,
+      address: BitGo.V2.TEST_ETH_WALLET_FIRST_ADDRESS,
+      tag: 'latest',
+    };
+    if (common.Environments[this.getEnv()].etherscanApiToken) {
+      params.apikey = common.Environments[this.getEnv()].etherscanApiToken;
+    }
+
+    // Nock tokens stuck on the wallet
+    nock('https://api-goerli.etherscan.io')
+      .get('/api')
+      .query(params)
+      .reply(200, { status: '1', message: 'OK', result: '2400' });
+
+    return wallet;
+  };
+
+  //
+  // registerToken
+  // Register tokens
+  //
+  BitGo.prototype.registerToken = function (tokenFamily: string, createTokenConstructor: CoinConstructor) {
```

Comment:
this is not necessary anymore now that we're decorating classes & each test is passing a starting reference. When we move tests over, they should use the registration methods from CoinFactory.